engine/src/main/resources/org/terasology/engine/assets/shaders/blockSelection.info

```diff
@@ -1,7 +1,7 @@
 {
     "params": [
         {
-            "name": "texture",
+            "name": "tex",
```

Comment:
Why? Is there some name clash with `texture`?

Reply:
yep you use texture instead of texture2d